all-repos.yaml

```diff
@@ -77,3 +77,4 @@
 - https://github.com/mattlqx/pre-commit-sign
 - https://github.com/mattlqx/pre-commit-search-and-replace
 - https://github.com/openstack-dev/bashate
+- https://github.com/alessandrojcm/commitlint-pre-commit-hook
```

Comment:
can't help but notice the irony that you didn't run `pre-commit` while committing this 😆